numpydoc/tests/test_docscrape.py

```diff
@@ -766,11 +767,10 @@ def test_warns():
     assert param.type == 'SomeWarning'
     assert param.desc == ['If needed']
 
-
-def test_see_also():
+@pytest.mark.parametrize('prefix', ['', '\n    '])
```

Comment:
this is a bit obscure, so please reference the present issue in a comment.

Reply:
Sure, done. This is also in the commit, so git blame will show the explanation. 

I've forced push but github does not show the new commit. I might close and reopen.